src/pages/Checkout/CheckoutPage.vue

```diff
@@ -362,12 +350,11 @@ export default {
 		},
 		showPayPal() {
 			return parseFloat(this.creditNeeded) > 0
-			&& (this.braintreeExpVersion === 'control' || this.braintree === false);
+			&& this.braintree === false;
 		},
 		showBraintree() {
 			return parseFloat(this.creditNeeded) > 0
-				&& this.braintree === true
-				&& this.braintreeExpVersion === 'shown';
+				&& this.braintree === true;
```

Comment:
I think we should probably remove the Ui setting for braintree and just set it to true always...